plugin/ensime.vim

```diff
@@ -38,6 +37,10 @@ command! -nargs=* -range EnToggleFullType call ensime#com_en_toggle_fulltype([<f
 command! -nargs=* -range EnOrganizeImports call ensime#com_en_organize_imports([<f-args>], '')
 command! -nargs=* -range EnAddImport call ensime#com_en_add_import([<f-args>], '')
 
+function! EnPackageDecl() abort
+  return ensime#com_en_package_decl()
+endfunction
```

Comment:
Typo here: you forgot to rename the `com_` to `fun_`.

I know you're following patterns that are already there, but this leads me to ask a question or two I've wondered about for awhile: do we need the boilerplate of this extra wrapper function, instead of just invoking `ensime#fun_en_package_decl()` directly? If we end up having to put tons of wrappers in `plugin/` then we won't get much benefit out of autoload.

Perhaps related, it always confuses me that we have many functions (and commands) taking `(args, range)` that don't actually use them. I assume without reading closely right this moment that this is baked into expectations of the `call_plugin` delegation, and possibly the `@execute_with_client()` decorator too. It seems like a smell to me, hard to understand and prone to maintenance headaches. It'd be a big refactoring to change this, and obviously out of scope for this pull request… Just wanted to bring it up since I keep wondering. Perhaps we can discuss/track it in a separate discussion or sync up on Gitter at some point.

In other news, it has finally occurred to me what the `com_` prefix means 😊